torch/_inductor/ir.py

```diff
@@ -1254,7 +1254,6 @@ def fn(index: int) -> OpsValue:
             isinstance(reduction_numel, Integer)
             and V.graph.sizevars.size_hint(reduction_numel)
             < config.unroll_reductions_threshold
-            and sympy_product(ranges) != 1
```

Comment:
intended?

Reply:
Yeah. I at least don't understand why this is needed 🤔  (cc: @jansel ?)